CMakeLists.txt

```diff
@@ -79,11 +79,25 @@ if (ENABLE_ALPINE)
     add_subdirectory (alpine)
 endif ()
 
-option (ENABLE_UNIT_TESTS "Enable unit tests" OFF)
+option (ENABLE_UNIT_TESTS "Enable unit tests" ON)
 if (ENABLE_UNIT_TESTS)
-    find_package (GTest REQUIRED HINT $ENV{GTEST_PREFIX})
-    add_subdirectory (unit_tests)
-    #add_subdirectory(unit_tests_single_precision)
+  include(FetchContent)
```

Comment:
Could you run the code formatting? The indent seems to be off compared to the other lines (see the above `if`) in this file.

Reply:
https://github.com/IPPL-framework/ippl/pull/233/commits/97c7b60d5e85396a3a62561c8706acc4dbc876a4

Resolved in this commit, however the automatic code formatting of `clang-format` is only working for `*.h, *.hpp` and `*.cpp` files.